.circleci/run.sh

```diff
@@ -7,8 +7,14 @@ if [ "$1" == "" ]; then
 fi
 
 circleci_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-found="$(grep -rn \"""$1"\"": packages/*/package.json | cut -d '/' -f2 | uniq)"
-modified="$(git diff origin/canary...HEAD --name-only | grep '^packages/' | cut -d '/' -f2 | uniq)"
+found="$(grep -rn "\"$1\":" packages/*/package.json | cut -d '/' -f2 | uniq)"
+modified="$(git diff origin/canary...HEAD --name-only | grep '^packages/' ||: | cut -d '/' -f2 | uniq)"
```

Comment:
These changes don't seem to work locally for me or in CI.

```
$ yarn test-unit
yarn run v1.17.3
$ ./.circleci/run.sh test-unit
The following packages were modified: 
1

Done in 0.09s.
```

The `modified` output is not cutting properly and still shows the full path instead of the package name.

Reply:
Let's change this from `.sh` to `.js` if possible (I'm not sure if `cd` works or if we can just set the cwd).